src/parser/view_transform.c

```diff
@@ -3226,7 +3228,7 @@ mq_copypush_sargable_terms_helper (PARSER_CONTEXT * parser, PT_NODE * statement,
 
 	  if (nullable_cnt)
 	    {
-	      continue;		/* do not copy-push nullable-term */
+	      continue;		/* do not copy-push nullable-term of outer spec */
```

Comment:
Is guaranteed that the nullable term is related to outer join?
I think that some term might not be related to the outer join and the term needs to be pushed.

Reply:
No. but If the spec is an outer join type, nullable terms no longer need to be checked for pushability.
Other specs( terms) will proceed in another mq_copypush_sargable_terms() call.

Reply:
Ok, I understood.